activejob/test/cases/exceptions_test.rb

```diff
@@ -154,36 +154,54 @@ class ExceptionsTest < ActiveSupport::TestCase
     ActiveJob::Base.retry_jitter = old_jitter
   end
 
-  test "retry jitter disabled with nil" do
-    travel_to Time.now
+  [
+    { klass: "DefaultsError", jitter: 0.6, text: "default" },
+    { klass: "ExponentialWaitTenAttemptsError", jitter: 1.2, text: "exponentially retrying" }
+  ].each do |error_info|
+    test "random wait time for #{error_info[:text]} job when retry jitter delay multiplier value is between 1 and 2" do
```

Comment:
Can we define two tests? There is not problem with duplicated code on tests.

Reply:
Thanks for the review! Sure, will add two tests for this

Reply:
Moved into separate test cases